internal/venice-common/src/main/java/com/linkedin/venice/meta/ReadOnlyStore.java

```diff
@@ -841,6 +844,232 @@ public void setPartitionerConfig(PartitionerConfig value) {
     throw new UnsupportedOperationException();
   }
 
+  public StoreProperties getStoreProperties() {
```

Comment:
Maybe rename this to `cloneStoreProperties`? To make it clear that we are returning a new StoreProperties.